src/linux/Packaging.Linux/shim.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+echo >&2 "warning: git-credential-manager-core was renamed to git-credential-manager"
+echo >&2 "warning: please update your credential.helper configuration"
```

Comment:
Do we want to include the command they should run to update?

Reply:
I asked about this, but I don't think we can be sure of the exact setup they have to be able to synthesise the right command. Did they update the system config? user config? `-c`? Envar?